src/utils/get-input-from-env.ts

```diff
@@ -57,6 +60,9 @@ const parseValue = (
     return parsed;
   }
   if (type === "boolean") {
+    if (value === "") {
+      return null;
+    }
```

Comment:
Similar fix here. We don't currently have any optional booleans though, so it doesn't really affect anything. But should help ensure we don't get caught out in future.